core/vm/actor/builtin/v0/verified_registry/verified_registry_actor.cpp

```diff
@@ -0,0 +1,223 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "vm/actor/builtin/v0/verified_registry/verified_registry_actor.hpp"
+#include "vm/actor/builtin/v0/verified_registry/verified_registry_actor_utils.hpp"
+
+namespace fc::vm::actor::builtin::v0::verified_registry {
+  namespace Utils = utils::verified_registry;
+
+  // Construct
+  //============================================================================
+
+  ACTOR_METHOD_IMPL(Construct) {
+    OUTCOME_TRY(runtime.validateImmediateCallerIs(kSystemActorAddress));
+
+    const auto id_addr = runtime.resolveAddress(params);
+    if (id_addr.has_error()) {
+      return ABORT_CAST(VMExitCode::kErrIllegalArgument);
+    }
```

Comment:
```suggestion
    REQUIRE_NO_ERROR(id_addr, VMExitCode::kErrIllegalArgument);
```

Reply:
REQUIRE_NO_ERROR could keep the original error, but here Lotus directly returns kErrIllegalArgument